browser/lib/util/http.js

```diff
@@ -3,6 +3,16 @@ var Http = (function() {
 
 	function Http() {}
 
+	function shouldFallback(err) {
+		var statusCode = err.statusCode;
+		/* 400 + no code = a generic xhr onerror. Browser doesn't give us enough
+		 * detail to know whether it's fallback-fixable, but it may be (eg if a
+		 * network issue), so try just in case */
+		return (statusCode === 408 && !err.code) ||
+			(statusCode === 400 && !err.code)      ||
```

Comment:
> Tabs are for leading indentation only; never anywhere except at the start of a line.

They were at the start of the line. Since the return statement was split over multiple lines, all except the first line were indented one level relative to the first, which was done with tabs. (But isn't any longer after the fixup)